src/stack_aggregator.py

```diff
@@ -591,8 +594,10 @@ def get_dependency_data(epv_set):
     epv_data = tr_epv_list['result']['data']
     epv_list, unknown_deps_list = find_unknown_deps(epv_data, epv_list,
                                                     tr_list, unknown_deps_list, True)
+    logger.info('Unknown Dependencies {ud}'.format(ud=unknown_deps_list))
```

Comment:
i think logs for unknown deps flow is already present in the utils function. i dont think its needed here again.

Reply:
Ack!